.gitignore

```diff
@@ -53,4 +54,11 @@ build
 .env
 .venv
 tsconfig.tsbuildinfo
+.yarn/unplugged
+.yarn/install-state.gz
+.yarn/cache
+.yarn/__virtual__
```

Comment:
Should we also ignore the yarn.lock file since now using pnpm? Best to remove that file as well if not already as could be missleading and cause future issues.